pkg/local_object_storage/shard/metrics_test.go

```diff
@@ -50,6 +50,10 @@ func (m metricsStore) DecObjectCounter(objectType string) {
 	m.AddToObjectCounter(objectType, -1)
 }
 
+func (m metricsStore) SetShardReadonly(readonly bool) {
+	m.SetShardReadonly(readonly)
```

Comment:
Infinite recursion here? Also let's make it `ReadOnly` -- the same way it is done in `mode` package.

Reply:
Thanks, fixed and test updated.